src/views/poi_bloc/hour.dot

```diff
@@ -3,12 +3,14 @@
     <p class="poi_panel__info__section__description">
       <div class="icon-icon_clock poi_panel__block__symbol"></div>
       <div class="poi_panel__block__content">
-        <div class="poi_panel__info__hours__status" {{= click(this.extend, this) }}>
-          {{= this.panel.renderPartial(this.nextTransitionPartial) }}
-          <i class="icon-icon_chevron-down poi_panel__info__hours__status__toggle"></i>
-        </div>
-        <div class="poi_panel__info__hours">
-          {{? !this.opening.isTwentyForSeven }}
+        {{? this.isTwentyFourSeven }}
+          <div class="poi_panel__info__hours__status__text">{{= _('Open 24h/24', 'hour block') }}</div>
```

Comment:
the status circle is missing here.  
Maybe this 24/7 logic should be part of "nextTransitionPartial" too.

Also, in english a more common wording would be "Open 24/7".